### DISCUSSION TITLE
授权过期，请求新的激活码，谢谢大佬

### DISCUSSION THREAD

**Discussion by xiaozzlx** [LFTEC/LearningFucker#52] [General]
请将下面代码复制到Github讨论区中，收到激活码后，调用activate命令进行激活
------------------------------License------------------------------
JwNf/mL9WenqEBoHg80YkzPjP7umBTIZw30l5V1gavzSzIUTpo+CDPeaUPiKuOh1KJu
B03tGPLuU6y7soClmQXMOsJkFS5QIVc+z1GUynSX3/2c3qcuOQ35B6iXIRssFCbv54a
kwkw2MysVlCCH5kYYgZ8Zl0vHws/9LF84PTMdAJ2clTmMQOtJHQBA1luoww3pUlfuB5
C6gKGhvu4tHOcOdFrLSnBYqEm/SU4br/FGZdHDUcT/AKIC5jHqL1ZAIVMuRSudH0cEa
un+z2MtYwFQAG68wrMMFJ2P9nDFQcF60OZhAxMc+7KOUh/nGfYFCn2mdegXKqhruMd/
Sw+aP/b0udcFBzXVdfttDUhnyV4d2KigrqZIZxVhjLkUcfYyfHPsImKmu3I6mt/3QNt
kP8ltmzUxU0s4h1R0oZlrW0Y4afoyMhj9RV0LigjcUusefzqLl9+NhKf2FYUk2GO9vU
OsxD082+NVEaT6uihG9noLfc6i3j8YeMiJl8HWCeuVz1OHMS4Zq+Z4oXk7Br+8OQeN2
NoceJG3HOuEK+M0+klRuZ1OeI59Wjn0VNiOOT7wSN6w3wRAboRWLu1HH80ffLsJ9WB7
u6CIzv8aLYoaFrti870f0wi/bH/hOkFSiYBjs5jy2a/Y8LVr2MaEbu3Ewp7rgvLmYqv
tY91RbH0UulgSpqMiYLkUDKogjfZ2FoMvrh9XZiQzyudFg6PkkUFLKhQQW5D8gxbycj
uCHo6zd2tCc/9ltAR9h1TG1g0eapDFhvTSilrPRTptbHWKaRplQSF6Jj86Pd/nvO9Ul
L/iZ6x7STuJlGT7UmniguRILJpxNrhKVw5b2OkyEa61+LTP38Q/lEDp1+gCO+hGHAD0
JqjVI87nTe+ZMzbTW8xNR8zfSBcyCS6y91jj8TUJu9J0kCbTHoieXwrVzpEqYa4NVF7
N0C1DCRWI+4DKraeydzUIEkXcMxTxdCMhsWhACaXtiUEfUavalsyrU+rGl6VFl8u55x
yHix934ZYKZuVX3Ch4cBU0gQYgTJ3azdndPfptYQfATFrSuhS6geOS57/Fg4Ta9DM6L
yIRPIOh29zcBDx68WzbI6QrLorlHsM85I688bOrRzFPkP8nBjnCI5TwkkP9m2x6etSB
jBkwL4NcWPr2gkCgmLzyGK5r9Pky/paQuODokt515B0MA+WVqDlubbAkbK+cv0EVNdA
Sn831Oq/gz8uAiw9DtesIrkGvN1L+Y5PB2Tcpk62O0ddC50vi8sPB8y2WgFhkZkybKa
tcU4UQANFO6atHXBu03s7LI6CJX5xIBTjfhVioNMgRQQDEQAAkNHGPKdBvxoPmVBy/G
9bGqhKP2uUBnKF6BuDYCjYObgA==
------------------------------License------------------------------

**Comment by LFTEC** [comment]
X1BeO0P/e+HRh5N9cjI5FpGAeVjiS/g39zN7O/ethZeS3fq+jb1v8FLWN674hRDeZRDppi5uxrVMi5iev5SDoo8apjXOOxJYshvGg8zymXAbLqNWSEqlCP2jQiSh8NGmro32eqBaezCor/bctTr7GHeRtIAoW2NcgqoRFDrgc6j6nPbufQFjBC22fZzgIvoNLVmUE4x65toE0um3a38fNCE70hT1jbiYD3rwo2UUJh3GHWzm+b8qq8QsAsI19q+KcwDLb9z5bhp1UhNC4qjIVLeEzZxaUkADCAblj8DEHsLa0CPkqFr5m0hKJh58TUIo5TggsEQnyFGfAI4JvTkoICQ0nKrrZBdBSA1POFFEjQKZ0gt0kouJIXiqjKt9T0zY6tkWERQzL5N+4YNoPcE61Or+JNz3EfB7AmTidlquFhkZuDGAUL0XwIjeFm/Qo7RW7H1Gc2VMckzA1b3vtai246obpS3WYSRgJxWDWTHOcJwF73Fvo7PnBKr7FkybbNz9oDcLBLqUBlrUTOZQxSdiaYcBQUCu4mnSbxCKdvAfpPILUtB3edN5stMv8DWW2cWRSYJWUPAHK2foTeNeY7vaDPAZviI/MzyohiebygvK+wWaRM4D8t68wz+ajPg/LIv0RkcS2Ufcihxwa8RGnsv6q3po3TuO2YRCEPvIRHU1GmCR1qp7m5rxpVSrcriPXwZkiv/l7AXoC7rG5p2GL3JIgfTuILacqkhDrDahF6aw6mU3P6xiVgCcPCBjYkQrBYyoZqj8/guC6myxIsOU2YmiJBA4LpoZFHxdihdY0QhAerZ7gMoJC8li86qX4xmFskdq+Cq6XTjy/N7ze7ZO+OooZ2a7OAy31TRPTu+dS3mc1Ibsur0YKHlCcy5LMhbRYOuuGYzJvaV2YdaUqaCuz2z6O/3pkQmzSedgjzx1AbOEY+Ro40PmsXoFByFIv4pM2sZwtkBIZucDFERGBpAkyDwKdwqOw3VJCdHAm1adLfvQyIXBQsrxgsmpWFrg/V+ujPVUnkeu8ojxCoW4EhL0Nb6TuENZI1ODl3QazBFweaIkAThYhzg8O9jiMykZJjhe4fbIrWLrEMWHiQobN6zJb0A1EVbOcEgVHnBsSh/HDWHGlnt3rQ5bnd1dngucmr7S3DQMdxE3wVWUMhM13kPfiNm2gvFyKGHH4nLrLLoCGDK0bysltkRtC0VIa6of/yFcB/Z/rv8tTIFj5zlIp/jWRok1ORhV96Va/fBlBJGr8LPkXRoEr1Fd3KkzntJ3k2Ga4uJ7QJZyOxU6tyXQ0kckdkME+Fnla5KNdAtqVTZD5YyQ5PgUUHd8/uAGBLzbdfPCThmK8BCv40XEu0hDVkG78JL/FA==
